### PR TITLE
Fix paths

### PR DESCRIPTION
**Pull request by ervanalb**

### CODE REVIEW

File: resources/qml/SettingsWidget.qml [hunk 1]
```diff
@@ -39,10 +39,9 @@ Item {
                 color: RadianceStyle.mainTextColor
             }
 
-            ComboBox {
-                id: modelNameComboBox
-                editable: true
-                model: ["gui", "cli"]
+            TextInput {
+                id: modelNameTextInput
+                text: "model.json"
```
Comment (zbanks) [inline resources/qml/SettingsWidget.qml]: What's the reasoning behind this change?

Comment (ervanalb) [reply resources/qml/SettingsWidget.qml]: I don't think there's ever a reason you would want to load the CLI model into radiance, and it felt silly to have a combobox with a single thing in it.

**Comment by ervanalb** [comment]
I want to move towards putting "models" into the library, and allowing you to instantiate them from the left sidebar. Since you can't quite do that right now, the workaround is that if you type in models in the right sidebar, it will load them from the library.

I wanted to split this functionality from the autosave and restore of the last radiance model. I think it would be weird and confusing if the autosave file showed up in the library, so I put it elsewhere. We can put additional config / state in this folder that is not necessarily a model, such as what MIDI controller you last used.

It's a little weird right now because the config folder only contains models, but I think it is moving in the right direction.

File: resources/qml/SettingsWidget.qml (hunk 1 above)
Comment (zbanks) [reply resources/qml/SettingsWidget.qml]: You would want to load it, edit it, and save it.

**Comment by zbanks** [comment]
You can merge this in

**Pull request merged**